index.md

```diff
@@ -0,0 +1,76 @@
+*  Such other matters related to the technical role of the TAC as may be communicated to the TAC by the Governing Board.
+
+# TAC Members
+
+All members of the broad ASWF community are welcome to participate in TAC activities, such as meetings or online discussion. For voting matters such as project inclusion or graduation, TAC members are composed of:
+
+* One representative appointed by each Premier Member;
+* One representative appointed by the technical oversight body (e.g., a technical steering committee) of each TAC Project; and
+* Up to three experts from the broader motion picture industry (who serve for one-year terms)
+
+Current TAC Members are:
+
+<table>
+    <tr>
+        <th>Representative</th>
+        <th>Appointed By</th>
+        <th>Role</th>
+        <th>Special Role</th>
+        <th>Organization</th>
+    </tr>
+{%- for member in site.data.tacmembers -%}
+    <tr>
+        <td>{{ member["Full Name"] }}</td>
+        <td>{{ member["Appointed By"] }}</td>
+        <td>{{ member["Voting Status"] }}</td>
+        <td>{{ member["Special Role"] }}</td>
+        <td>{{ member["Account Name: Account Name"] }}</td>
+    </tr>
+{%- endfor -%}
+</table>
+
+# Technical Projects
+
+Technical Projects are approved by TAC Members per the [ASWF Project Lifecycle guidelines](https://github.com/AcademySoftwareFoundation/tac/blob/master/process/lifecycle.md). You can submit your project for inclusion as an ASWF Technical Project by completing the [Project Contribution Proposal Template](https://github.com/AcademySoftwareFoundation/tac/blob/master/process/proposal_template.md) and emailing it to tac@lists.awsf.io.
+
+Each of the ASWF projects are open to participation by anyone subject to the governance each project has adopted. If you are looking for a way to contribute to a project, many ASWF projects maintain a list of "good first issues" ( linked to below ).
+
+Current Technical Projects are listed below.
+
+<!-- Embed list of all Open Mainframe Project members -->  
+<iframe src="https://landscape.aswf.io/pages/hosted-projects" frameborder="0" id="landscape" scrolling="no" style="width: 1px; min-width: 100%; opacity: 1; visibility: visible; overflow: hidden; height: 1717px;"></iframe>
+<script src="https://landscape.aswf.io/iframeResizer.js"></script>
+
+
+There are many more VFX related open source projects than what is hosted at ASWF; check out the list and add any we are missing at the [ASWF Landscape](https://landscape.aswf.io)
+
+[![ASWF Landscape](https://landscape.aswf.io/images/landscape.png)](https://landscape.aswf.io)
+
+# Current Working Groups
+
+Working groups may be formed by the TAC to focus on specific areas of technical interest or need within the ASWF.  They report to the TAC which remains responsible for any formal decisions or voting matters.
+
+## Active
+
```

Comment:
There are some missing WGs:

- Review and Approval
- Diversity and Inclusion
- Assets

Reply:
I'll add D&I, but the other two haven't been approved yet so technically aren't a WG yet.